spec/controllers/account_reset/delete_account_controller_spec.rb

```diff
@@ -100,6 +104,96 @@
         t('errors.account_reset.granted_token_expired', app_name: APP_NAME),
       )
     end
+
+    it 'logs info about user verified account' do
+      user = create(:user, :proofed)
+      create_account_reset_request_for(user)
+      grant_request(user)
+      session[:granted_token] = AccountResetRequest.first.granted_token
+
+      delete :delete
+
+      expect(@analytics).to have_logged_event(
+        'Account Reset: delete',
+        user_id: user.uuid,
+        success: true,
+        errors: {},
+        mfa_method_counts: { phone: 1 },
+        profile_idv_level: 'legacy_unsupervised',
+        identity_verified: true,
+        account_age_in_days: 0,
+        account_confirmed_at: user.confirmed_at,
+      )
+    end
+
+    it 'logs info about user biometrically verified account' do
+      user = create(
+        :user, :proofed_with_selfie, :with_phone
+      )
+      create_account_reset_request_for(user)
+      grant_request(user)
+      session[:granted_token] = AccountResetRequest.first.granted_token
+
+      delete :delete
+
+      expect(@analytics).to have_logged_event(
+        'Account Reset: delete',
+        user_id: user.uuid,
+        success: true,
+        errors: {},
+        mfa_method_counts: { phone: 1 },
+        profile_idv_level: 'unsupervised_with_selfie',
+        identity_verified: true,
+        account_age_in_days: 0,
+        account_confirmed_at: user.confirmed_at,
+      )
+    end
+
+    it 'logs info about user with a verified by mail account' do
+      user = create(:user, :proofed_with_gpo)
+      create_account_reset_request_for(user)
+      grant_request(user)
+      session[:granted_token] = AccountResetRequest.first.granted_token
+
+      delete :delete
+
+      expect(@analytics).to have_logged_event(
+        'Account Reset: delete',
+        user_id: user.uuid,
+        success: true,
+        errors: {},
+        mfa_method_counts: { phone: 1 },
+        profile_idv_level: 'legacy_unsupervised',
+        identity_verified: true,
+        account_age_in_days: 0,
+        account_confirmed_at: user.confirmed_at,
+      )
+    end
+
```

Comment:
I think we could drop this since we're not expecting anything distinguishable from GPO anymore.

```suggestion
```

Reply:
That makes sense, thank you!